### PR TITLE
Create project model

### LINKED ISSUE #7: Project
ID
title
description
project.url
github.url

### PR DESCRIPTION
**Pull request by TheSabby**
Closes #7

### CODE REVIEW

File: models/project.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  var Project = sequelize.define('Project', {
+    title: {
+      allowNull: false,
+      type: Sequelize.STRING
+    },
+    description: {
+      allowNull: true,
```
Comment (the-vampiire) [inline models/project.js]: should we allow a null description? i think at least a line or two is necessary to define a project

File: models/project.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      type: Sequelize.TEXT
+    },
+    project_url: {
+      allowNull: true,
+      type: Sequelize.STRING
+    },
+    github_url: {
+      allowNull: true,
```
Comment (the-vampiire) [inline models/project.js]: same for the github url @TheSabby

File: models/project.js (hunk 1 above)
Comment (TheSabby) [reply models/project.js]: Fixed

File: models/project.js (hunk 2 above)
Comment (TheSabby) [reply models/project.js]: Fixed

File: models/project.js [hunk 3]
```diff
@@ -3,19 +3,19 @@ module.exports = (sequelize, DataTypes) => {
   var Project = sequelize.define('Project', {
     title: {
       allowNull: false,
-      type: Sequelize.STRING
+      type: DataTypes.STRING
     },
     description: {
       allowNull: true,
-      type: Sequelize.TEXT
+      type: DataTypes.TEXT
```
Comment (the-vampiire) [inline models/project.js]: do we need to pass the length here or will that be assigned based on the user's input length?

**DataTypes.TEXT Documentation**
> An unlimited length text column. Available lengths: tiny, medium, long
http://docs.sequelizejs.com/variable/index.html#static-variable-DataTypes

Comment (TheSabby) [reply models/project.js]: TEXT is a datatype in SQL which is unlimited. Specifying a length is optional

**Pull request merged**